test/nodetool/README.md

```diff
@@ -0,0 +1,23 @@
+# Front-end tests for nodetool
+
+The tests in this directory excercise the nodetool client itself, mocking the API backend.
+This allows for testing all combinations of all supported options, and still keeping the tests quick.
+
+The tests can be run against both the scylla-native nodetool (default), or the inherited, C\*-based nodetool.
+
+Run all tests against the scylla-native nodetool:
+```
+pytest --nodetool=scylla .
+```
+
+You can specify the path to the scylla binary with the `--nodetool-path` option. By default the tests will pick up the ScyllaDB executable, that is appropriate for the `--mode` option (defaults to `dev`).
+
+Run all tests against the C* nodetool:
+```
+pytest --nodetool=cassandra .
+```
+
+Again, you can specify the path to the nodetool binary with `--nodetool-path` option. By default, `scylladb.git/tools/java/bin/nodetool` will be used.
```

Comment:
`scylladb.git` stands for the root directory of ... `scylladb.git`. I didn't want to use relative paths, because this README.md is in the `tests/nodetool` directory and I don't know how the reader would interpret relative paths.

Reply:
Ok.

Reply:
I changed this to `<scylladb.git>` so it is more clear, I'm talking about the root of the repo.